torch/fx/experimental/proxy_tensor.py

```diff
@@ -384,6 +384,7 @@ def create_arg(self, a: Any):
 
             return self.create_node('get_attr', qualname, (), {})
         elif isinstance(a, (SymInt, SymFloat)):
+            breakpoint()
```

Comment:
Remove

Reply:
There's a weird failure here I was debugging. For some reason the assert triggers 🤔